web/docs/tutorial/07-auth.md

```diff
@@ -271,33 +269,41 @@ wasp db studio
 ```
 
 <img alt="Database demonstration - password hashing"
-src={useBaseUrl('img/wasp_db_hash_demonstration.gif')}
+src={useBaseUrl('img/wasp_user_in_db.gif')}
 style={{ border: "1px solid black" }}
 />
 
-We see there is a user and that its password is already hashed 🤯
+
+You'll notice that we now have a `User` entity in the database alongside the `Task` entity.
 
 However, you will notice that if you try logging in as different users and creating some tasks, all users share the same tasks. That's because we haven't yet updated the queries and actions to have per-user tasks. Let's do that next.
 
+<small>
```

Comment:
The auth entities are not relevant for the tutorial, but they will see them, so I thought it's best to address their existence.